src/plugins/intel_cpu/src/nodes/executors/acl/acl_lowp_fullyconnected.cpp

```diff
@@ -0,0 +1,143 @@
+#include "nodes/executors/acl/acl_utils.hpp"
+#include "nodes/executors/executor.hpp"
+#include "nodes/executors/memory_arguments.hpp"
+#include "nodes/executors/debug_messages.hpp"
+#include "nodes/executors/implementation_utils.hpp"
+#include "utils/debug_capabilities.h"
+
+namespace ov {
+namespace intel_cpu {
+
+static bool checkPostOps(const PostOps &postOps) {
+    if (postOps.empty()) {
+        return true;
+    }
+
+    if (postOps.size() != 1) {
+        return false;
+    }
+
+    const auto activation = std::dynamic_pointer_cast<ActivationPostOp>(postOps[0]);
+    return checkActivationLayerInfo(convertToEltwiseAlgorithm(activation->type()));
+}
+
+static void initFCAttrs(const FCAttrs &attrs,
+                        ACLTensorAttrs& aclTensorAttrs,
+                        ACLFCAttrs& aclfcAttrs,
+                        const MemoryArgs &memory,
+                        arm_compute::GEMMInfo& fullyConnectedLayerInfo,
+                        const PostOps &postOps) {
+    aclTensorAttrs.hasLayoutTypeNHWC = memory.at(ARG_SRC)->getDescPtr()->hasLayoutType(LayoutType::nspc);
+    aclfcAttrs.inputPrecision = memory.at(ARG_SRC)->getDescPtr()->getPrecision();
+    aclfcAttrs.weightsNonTransposed = attrs.weightsNonTransposed;
+
+    if (!postOps.empty()) {
+        auto activation = std::dynamic_pointer_cast<ActivationPostOp>(postOps[0]);
+        fullyConnectedLayerInfo.set_activation_info(getActivationLayerInfo(
+                convertToEltwiseAlgorithm(activation->type()),
+                activation->alpha(), activation->beta(), activation->gamma()));
+    }
+
+    if (memory.at(ARG_SRC)->getPrecision() != memory.at(ARG_WEI)->getPrecision()) {
+        aclfcAttrs.isConvertedWeights = true;
+    }
+}
+
+ACLLowpFullyConnectedExecutor::ACLLowpFullyConnectedExecutor(const FCAttrs &attrs,
+                                                             const PostOps &postOps,
+                                                             const MemoryArgs &memory,
+                                                             const ExecutorContext::CPtr& context) : dequantizationScales(attrs.dequantizationScales) {
+    initFCAttrs(attrs, aclTensorAttrs, aclfcAttrs, memory, gemmInfo, postOps);
+    packedWeights = acl_fc_executor::prepareWeightMemory(memory, context, attrs, aclfcAttrs, postOps);
+}
+
+bool ACLLowpFullyConnectedExecutor::supports(const FCConfig &config) {
+    const auto src0 = srcType(config);
+    const auto src1 = weiType(config);
+    const auto dst = dstType(config);
+    if ((src0 != ov::element::i8) || (src1 != ov::element::i8) || (dst != ov::element::f32)) {
+        return false;
+    }
+
+    VERIFY(checkPostOps(config.postOps), UNSUPPORTED_TYPE_OF_POSTOPS);
+    VERIFY(one_of(srcRank(config), 2U, 3U, 4U), UNSUPPORTED_SRC_RANK);
+    VERIFY(one_of(weiRank(config), 2U, 3U, 4U), UNSUPPORTED_WEI_RANK);
+    VERIFY(static_cast<FCAttrs>(config.attrs).dequantizationScales.size() <= 1, UNSUPPORTED_PER_CHANNEL_QUANTIZATION);
```

Comment:
Why do we expect that dequantizationScales can be empty?
Is it possible that quantized FullyConnected will not contain dequantization scales at all?

Reply:
If it's empty default scale is used: `1.0`.

Reply:
It looks like dequantizationScales can be empty, since there is the same empty vector check in https://github.com/openvinotoolkit/openvino/blob/6489755e9f91fbb8bb273ee781f80494570f42cb/src/plugins/intel_cpu/src/dnnl_postops_composer.cpp#L44

Reply:
The vector is resized to contain at least one element in `Node::fuseDQScales`:
https://github.com/openvinotoolkit/openvino/blob/6489755e9f91fbb8bb273ee781f80494570f42cb/src/plugins/intel_cpu/src/node.cpp#L1874-L1875

Reply:
@EgorDuplensky some cases when Multiply is not fused as dequantization scales were discussed on the sync (e.g. FC without bias but with activation right after FC: in this case Multiply is propagated through the activation, and is not fused as DQ scale)

Reply:
@EgorDuplensky case FC with bias and activation is added. In this case dequantization scales, bias, activation are fused into FC.